src/app/test-runners/playwright/record-your-first-replay/page.md

```diff
@@ -10,33 +10,37 @@ title: Playwright
 {% tab %}
 
 ```sh
-npm install @replayio/playwright -D
+npm install @replayio/playwright@alpha -D
 ```
 
 {% /tab %}
 {% tab %}
 
 ```sh
-yarn add @replayio/playwright -D
+yarn add @replayio/playwright@alpha -D
 ```
 
 {% /tab %}
 {% tab %}
 
 ```sh
-pnpm install @replayio/playwright -D
+pnpm install @replayio/playwright@alpha -D
 ```
 
 {% /tab %}
 {% tab %}
 
 ```sh
-bun install @replayio/playwright -D
+bun install @replayio/playwright@alpha -D
 ```
 
 {% /tab %}
 {% /tabs %}
 
+{% callout title="Note" type="note" %}
+We recommend using the current `@alpha` version of this plugin. It's more robust and better but we are still polishing it. If you encounter any problems with it you can use `@replayio/playwright@alpha`. We'd appreciate feedback and bug reports to be reported [here](https://github.com/replayio/replay-cli/issues/new).
```

Comment:
we should be recommending using `@latest` as the fallback, yes?

```suggestion
We recommend using the current `@alpha` version of this plugin. It's more robust and better but we are still polishing it. If you encounter any problems with it you can use `@replayio/playwright@latest`. We'd appreciate feedback and bug reports to be reported [here](https://github.com/replayio/replay-cli/issues/new).
```

Reply:
oh, good catch 😅